galley/docker/testing/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#
+# To test/run the image manually, use this command-line:
+# docker run -it --mount type=tmpfs,target=/app,tmpfs-mode=1770 --expose=8080  -p 8080:8080 <imageid>
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+SCRATCH_DIR="/tmp/galley/test/docker"
+
+K8S_VER="${K8S_VER:-v1.9.2}"
```

Comment:
I think I've seen those already in other scripts (init ? or localenv*sh) - cc @costinm